doc/man3/EVP_EncodeInit.pod

```diff
@@ -125,6 +127,11 @@ output will be padded with 0 bits if necessary to ensure that the output is
 always 3 bytes for every 4 input bytes. This function will return the length of
 the data decoded or -1 on error.
 
+EVP_DecodeBlock_ex() is an extension of EVP_DecodeBlock(), which won't count in
```

Comment:
IMO the semantics does not make much sense to me. The EVP_ENCODE_CTX argument is IMO useless. Why not add some flag option to the evp_decodeblock_int() and use that flag argument instead?

Reply:
Hi Tomáš,
Yeah, that will be more straightforward!

Since EVP_DecodeBlock is de-facto obsolete, I think I should go with general API.